script/Ciel/Day 7/07_01_CIEL07_6.txt

```diff
@@ -777,46 +801,46 @@ All of a sudden, Arcueid is back, walking amidst the sea of blood.
 [sha:accd078b22c417d24b1f4c47a91e521e13ce511e]{
 -- Page 552, Offset 16135.
 -- 散乱した肉片はサラサラと灰になって風に散っていく。
-The scattered flesh turns to ash, which is then swept up by the wind.
+The scattered flesh crumbles into ash, and is swept away by the wind.
 }
 [sha:2075b2b0daa371c79ead281c5b05dc51c96d0a3c]{
 -- Page 552, Offset 16136.
 -- 残ったものはオイルのような血だまりだけ。
-Only the oily bloodstains remain.
+All that remains is the blood, pooling like so much oil.
```

Comment:
I like the previous version.

Reply:
Reason I changed this one was because オイルのような is "oil-like" (thick, viscous, staining) but "oily" initially read to me as "infused, mixed with or covered in oil" which gives the wrong impression (worsened somewhat by the tendency for 油 to be used around gore). If you don't think that misread is an issue, or have a better suggestion, happy to change it.

Reply:
I personally don't think that's much of an issue. In fact, IIRC DA blood is often described with this quality, so it's a rather literal thing.